neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
OK, that is a better name for the section_length (=sum(length of segments in a section) indeed. However, that is not what path distance should be. The path distance is supposed to be the sum of all the segment lengths from a section to the root (so the section-length plus all its ancestors section-lengths).